### ISSUE TITLE
Import name

### ISSUE THREAD

**Issue by notchris** [cyrilf/vue-dat-gui#5]
Hello! This is a great port, thank you for making it.

I found that I could only import the module via: 
`import DatGui from '@cyrilf/vue-dat-gui';`
while the documentation says to use
`import DatGui from 'vue-dat-gui';`

- Chris

**Comment by notchris** [comment]
Saw its listed  correctly on the github, this is only on npm.

**Comment by cyrilf** [comment]
Hey @notchris 
Good catch :v: 
There was one typo left in the README. But yeah, npm README is still outdated, I bumped the patch version and published it. I guess now it's only a matter of time before it gets updated.
Thanks!

**Comment by cyrilf** [comment]
Thanks for the feedback.
It's available in the latest version, `v0.3.0` I added a prop `closePosition` that defaults to "bottom" but you can set it up to "top".
You can use it like:

```
<dat-gui closeText="Close controls" openText="Open controls" closePosition="top">
  ...
</dat-gui>
```

I added it because it's easy to do. But I don't want to deviate too much in terms of functionality from what `dat-gui` are providing. Feel free to open a PR next time :v:

**Comment by notchris** [comment]
Ah thank you! And I agree. I'll make sure to make a PR. Thanks again!
